doc/radosgw/notifications.rst

```diff
@@ -17,27 +17,27 @@ user can only manage its own topics, and can only associate them with buckets it
 
 In order to send notifications for events for a specific bucket, a notification entity needs to be created. A
 notification can be created on a subset of event types, or for all event types (default).
-The notification may also filter out events based on preffix/suffix and/or regular expression matching of the keys. As well as, 
+The notification may also filter out events based on prefix/suffix and/or regular expression matching of the keys. As well as,
 on the metadata attributes attached to the object.
 There can be multiple notifications for any specific topic, and the same topic could be used for multiple notifications.
 
 REST API has been defined to provide configuration and control interfaces for the bucket notification
-mechanism. This API is similar to the one defined as S3-compatible API of the pubsub sync module.
+mechanism. This API is similar to the one defined as the S3-compatible API of the pubsub sync module.
 
 .. toctree::
    :maxdepth: 1
 
    S3 Bucket Notification Compatibility <s3-notification-compatibility>
 
-Notificatios Performance Stats
+Notification Performance Stats
 ------------------------------
-Same counters are shared between the pubsub sync module and the bucket notification mechanism.
+The same counters are shared between the pubsub sync module and the bucket notification mechanism.
 
-- ``pubsub_event_triggered``: running counter of events with at lease one topic associated with them
+- ``pubsub_event_triggered``: running counter of events with at least one topic associated with them
 - ``pubsub_event_lost``: running counter of events that had topics associated with them but that were not pushed to any of the endpoints
 - ``pubsub_push_ok``: running counter, for all notifications, of events successfully pushed to their endpoint
 - ``pubsub_push_fail``: running counter, for all notifications, of events failed to be pushed to their endpoint
-- ``pubsub_push_pending``: gauge value of events pushed to an endpoint but not acked or nacked yet
+- ``pubsub_push_pending``: gauge value of events pushed to an endpoint but not acked or acked yet
```

Comment:
this is not a type. nack means negative ack.

Reply:
changed back